src/test/java/com/teragrep/pth10/evalTest.java

```diff
@@ -2343,49 +2321,26 @@ public void parseEvalStrptimeCatalystTest() {
             assertEquals(1534184553L, lstB.get(0).getLong(0));
         });
     }
-    
-    // FIXME Test eval strftime()
-    @Disabled
-	@Test
+
+    @Test
+    @DisabledIfSystemProperty(named="skipSparkTest", matches="true")
     public void parseEvalStrftimeCatalystTest() {
-	     /*String q = "index=index_A <!--| eval a=strftime(1534159353,\"%Y-%m-%d %H:%M:%S\") " +
-	 			   "| eval b=strftime(1534188153,\"%Y-%m-%d %T %I %p %Z\") --> | eval c=strftime(_time, \"%Y-%m-%d %H:%M:%S\")";
+        String q = "index=index_A | eval a=strftime(1534159353,\"%Y-%m-%d %H:%M:%S\") " +
+                "| eval b=strftime(1534188153,\"%Y-%m-%d %T %I %p %Z\")";
```

Comment:
I think there should also be a test where the epoch is taken from a field like there was in the commented out test.

like `strftime(_time, ...)`

Reply:
Fixed in [the new commit](https://github.com/teragrep/pth_10/pull/316/commits/53f58b2e7bdf82c27f93b435334c5a73e736ea42)